### PR TITLE
Fix include guards

### PR DESCRIPTION
**Pull request by moatom**
I fixed include guards using the identifiers reserved by C++.

### CODE REVIEW

**Review by gottesmm — APPROVED** [review]
Looks great! Thanks! = ).

**Comment by gottesmm** [comment]
@swift-ci test and merge

**Comment by compnerd** [comment]
@swift-ci please test and merge

**Comment by moatom** [comment]
Sorry.
I accidentally send this PR without testing in my environment.
But I didn't think it failed.

Anyway, it seems to work fine if undoing the changes of HeapObject.h, KeyPath.h and System.h's  include guards.
I am investigating exact reasons.

**Comment by moatom** [comment]
Now, It would work.
Please check.

**Comment by CodaFi** [comment]
@swift-ci please test

**Comment by CodaFi** [comment]
@swift-ci please test

**Comment by CodaFi** [comment]
@swift-ci please test

**Comment by moatom** [comment]
@gottesmm 
Could you merge it, or is there something wrong with it?

File: stdlib/public/SwiftShims/Target.h [hunk 1]
```diff
@@ -31,4 +31,4 @@
 # endif
 #endif
 
-#endif /* SWIFT_ABI_TARGET_H */
+#endif /* SWIFT_STDLIB_SHIMS_ABI_TARGET_H */
```
Comment (gottesmm) [inline stdlib/public/SwiftShims/Target.h]: Can you change /* to //?

**Comment by moatom** [comment]
ok

**Comment by compnerd** [comment]
@moatom could you please squash the commits into a single commit?

**Comment by moatom** [comment]
@compnerd 
Thank you for pointing it out.
It is `rebase -i`, isn't it?

I have another question.
When I searched Swift's source code by `#endif /* `, I found further ununified comments at the namespace ends in 24 files.
Should I change their `/*` to `//` this time?
We cannot avoid recompilation of all the translation units depending on them :(.

**Comment by compnerd** [comment]
Yes, use `git rebase -i` to squash the commits.

Sure, changing the ending on those in a follow up is fine.

**Comment by moatom** [comment]
Done!

**Comment by compnerd** [comment]
@swift-ci please test

**Comment by compnerd** [comment]
@swift-ci please test

**Pull request merged**